### DISCUSSION TITLE
GitHub Student Developer Pack Approved, but Benefits Not Activated After 8 Days

### DISCUSSION THREAD

**Discussion by usmanjutt47** [community/community#147929] [GitHub Education] [answered]
![Screenshot_20241226-061831.png](https://github.com/user-attachments/assets/59601bf8-e1c2-4591-943e-fda6f3ac95e2)

GitHub Student Developer Pack benefits. My application was initially rejected, but it was later approved on December 19, 2024. However, even after 8 days, access to the benefits, including GitHub Copilot.

**Comment by davevad93** [comment, marked as answer]
Hi @usmanjutt47 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @usmanjutt47 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by usmanjutt47** [reply]
Is this issue happening because my application was initially rejected and later accepted?

**Reply by davevad93** [reply]
Hey @usmanjutt47, no it's not due to that, the waiting time isn't the same for all users and may vary, but don't worry you'll get your benefits soon! 👍
